jdk/make/CompileLaunchers.gmk

```diff
@@ -449,8 +449,12 @@ ifeq ($(OPENJDK_TARGET_OS), windows)
 endif
 
 $(eval $(call SetupLauncher,jps, \
-    -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "openj9.tools.attach.diagnostics.Jps" }' \
-    -DAPP_CLASSPATH='{ "/lib/tools.jar" }'))
+    -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "openj9.tools.attach.diagnostics.tools.Jps"$(COMMA) }' \
```

Comment:
Please remove the trailing `$(COMMA)`'s, they are not needed.

Reply:
The unnecessary `$(COMMA)`s are still here.